queue_services/common/src/entity_queue_common/service_utils/service_logger.py

```diff
@@ -29,6 +29,10 @@ def before_breadcrumb(crumb, hint):  # pylint: disable=unused-argument; callback
 
 # Configure Sentry
 SENTRY_DSN = os.getenv('SENTRY_DSN', None)
+
+# Configure default log level to ERROR
+# Override with envirionment variable if needed
+LOG_LEVEL = os.getenv('LOG_LEVEL', logging.ERROR).upper()
```

Comment:
just a note that I think there are queue services that are not in the LEAR repo that are referencing this service logger module.  This will mean log levels will be set to `ERROR` if they don't have the `LOG_LEVEL` config right?

Reply:
That's correct.  To change the log level in any container / pod, add "LOG_LEVEL" to the appropriate container and environment in 1password.  Once saved, redeploy the container and the environment variable will automatically be added.  Please let me know if you have ANY concerns.

Reply:
That's fine as long as we let the other teams know about it.  I think at least a bunch of the auth listeners in other repos are using this common service logger.  The next time they push deploy through the different environments they will have unexpected log levels if they are unaware.  Maybe give a heads up in developer channel after this change makes it into `main`.

For example, this listener in the auth repo is using this logger.  https://github.com/bcgov/sbc-auth/blob/7bf026b76e5094b1b9f5f6aef68b073d918fcd2c/queue_services/business-events-listener/requirements.txt#L34